pkg/cmd/release/create/http.go

```diff
@@ -189,12 +189,16 @@ func createRelease(httpClient *http.Client, repo ghrepo.Interface, params map[st
 	return &newRelease, err
 }
 
-func publishRelease(httpClient *http.Client, releaseURL string, discussionCategory string) (*shared.Release, error) {
+func publishRelease(httpClient *http.Client, releaseURL string, discussionCategory string, isLatest *bool) (*shared.Release, error) {
 	params := map[string]interface{}{"draft": false}
 	if discussionCategory != "" {
 		params["discussion_category_name"] = discussionCategory
 	}
 
+	if isLatest != nil {
+		params["make_latest"] = *isLatest
+	}
```

Comment:
```suggestion
	if isLatest != nil {
		params["make_latest"] = fmt.Sprintf("%v", *isLatest)
	}
```

Lets copy what is being done [here](https://github.com/cli/cli/blob/7738b6187c80f128728aeba15fdba6bf62ec9e1b/pkg/cmd/release/create/create.go#L421C1-L424C3) for consistency.